tests/api_data/endpoint_data.json

```diff
@@ -357,7 +357,7 @@
       "type_of_application": "sporting",
       "project_name": "Save the rugby pitch",
       "funding_amount_requested": "25000",
-      "location": "Scotland"
+      "location_json_blob": "Scotland"
```

Comment:
Shouldn't this be JSON data rather than a string?

Reply:
Good spot - fixed accordingly